doc/ref_model/chapters/chapter03.md

```diff
@@ -282,7 +282,22 @@ The terms Container Infrastructure Service Instance and Container Infrastructure
 ## 3.8 Hardware Acceleration Abstraction
 
 <a name="3.8.1"></a>
-### 3.8.1 Types of accelerators
+### 3.8.1 Types of Accelerators
+
+Accelerator technologies can be categorized depending on where they are realized in the hardware product and how they get activated, life cycle managed and supported in running infrastructure.
+
+| Acceleration technology/hardware | Example implementation | Activation/LCM/support | Usage by application tenant |
+|---|---|---|---|
+| CPU instructions | Within CPU cores | None for hardware | Could be application binary differences (application to load software library that recognizes and uses) |
```

Comment:
"Could be application binary differences" would refer to with and without the additional CPU instructions. The column title, however, refers to "Usage". Should this just state what is currently in the parentheses and the outcome "application binary differences between versions that use and do not use these instructions" can follow.

Reply:
> 
> 
> "Could be application binary differences" would refer to with and without the additional CPU instructions. The column title, however, refers to "Usage". Should this just state what is currently in the parentheses and the outcome "application binary differences between versions that use and do not use these instructions" can follow.

Yes, changed. Thanks